main/performance.cpp

```diff
@@ -208,6 +208,38 @@ Performance::MonitorType Performance::get_monitor_type(Monitor p_monitor) const
 	return types[p_monitor];
 }
 
+Performance::MonitorDirection Performance::get_monitor_direction(Monitor p_monitor) const {
+	ERR_FAIL_INDEX_V(p_monitor, MONITOR_MAX, MONITOR_DIRECTION_NEUTRAL);
+	// ugly
```

Comment:
Indeed :(

If there's only one which is not `LOWER_IS_BETTER` do we really need all this boilerplate, instead of hardcoding the special case?